src/components/app/App.tsx

```diff
@@ -60,6 +61,7 @@ export class Evergreen extends React.Component<Props, State> {
             <Route path="/admin" render={admin} />
             <Route path="/patches" render={patches} />
           </div>
+          <ConfigDrop updateClientConfig={this.updateConfig}/>
```

Comment:
Embedding this here will make it show up on all pages won't it? I think we want a specific place the user goes to set the config, or possibly hiding this once we have a config

Reply:
That is correct. When we talked about this initially, we talked about putting it on the admin page (which is why it's currently in the admin folder), but we ran in to that issue with the routing to that page so we decided to just throw it on the main app page instead. Where do you think it should go now, since I'm still having that problem locally?

Reply:
One idea is to make it a modal that you can get to from the menu (but the menu option only shows if we have no config). The main thing is to not show it on every page

Reply:
Just discovered that the admin page does load for me locally now (not sure what changed), so I moved the config drop zone to that page.